src/core/Akka/Routing/Router.cs

```diff
@@ -187,14 +187,15 @@ public class Router
         //We need one unique constructor to handle this call: new Router(logic). The other constructor will handle that.
         //So in order to not confuse the compiler we demand at least one ActorRef. /@hcanber
         public Router(RoutingLogic logic, IActorRef routee, params IActorRef[] routees)
-        {
-            var routeesLength = routees.Length;
```

Comment:
If an explicit {{null}} is being passed as {{routees}}, the {{.Length}} will throw a NRE.